extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -467,6 +477,37 @@ public void setPrincipalClaim(String principalClaim) {
         }
     }
 
+    @ConfigGroup
+    public static class Proxy {
+
+        /**
+         * The host (name or IP address) of the Proxy.<br/>
+         * Note: If OIDC adapter needs to use a Proxy to talk with OIDC server (Provider),
+         * then at least both "host" and "port" items must be configured to enable the usage of a Proxy.
+         */
+        @ConfigItem
+        public Optional<String> host = Optional.empty();
+
+        /**
+         * The port number of the Proxy.
+         */
+        @ConfigItem(defaultValue = "80")
+        public OptionalInt port = OptionalInt.empty();
```

Comment:
Not being picky here, but you can also use `OptionalInt.of(80)` for cases where config injection is not available :wink:

Reply:
Or maybe replace the OptionalInt with an `int`, since we're assuming a default value now

Reply:
No, it's fine. 😊 You're right.

Reply:
So, woud this work:
```java
@ConfigItem
public int port = 80;
```

Reply:
Can we clarify it here on the conversation below, please?